charts/spire/examples/production/your-values.yaml

```diff
@@ -0,0 +1,20 @@
+# These values should be in your own configuration values.yaml
```

Comment:
I would prefer we had comments, perhaps with "# CHANGE_ME: " labels on what to update, so we can give guidance on what to set